.github/workflows/release.yml

```diff
@@ -35,16 +35,12 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        golang_cross: [ 1.16, 1.16-el7 ]
+        golang_cross: [ 1.19-bullseye ]
         include:
-          - golang_cross: 1.16-el7
-            goreleaser: 'ci/goreleaser/goreleaser-el7.yml'
-            rpmvers: 'el/7'
-            debvers: 'ubuntu/xenial ubuntu/bionic debian/jessie'
-          - golang_cross: 1.16
+          - golang_cross: 1.19-bullseye
             goreleaser: 'ci/goreleaser/goreleaser.yml'
-            rpmvers: 'el/8'
-            debvers: 'ubuntu/focal debian/buster debian/bullseye'
```

Comment:
ubuntu/bionic can be added here unless there is a problem with it.

Reply:
Added all the rpmvers/debvers like on /tyk (el/8, etc.), should be consistent.